src/rightClick.js

```diff
@@ -0,0 +1,39 @@
+chrome.runtime.onInstalled.addListener(function() {
```

Comment:
Make a query that asks for these when a new window is created
https://developer.chrome.com/docs/extensions/reference/windows/#event-onCreated